docs/iris/src/whitepapers/change_management.rst

```diff
@@ -0,0 +1,393 @@
+      same, in any subsequent sub-release with the same major release number.
+
+    * The only time we will make changes that can break existing code is at
+      a **major release**.
+
+    * At a major release, code that works **and emits no deprecation warnings**
+      in the latest previous (minor) release should still work, and behave
+      exactly the same.
+
+
+**What can possibly go wrong ?**
+
+If your code produces :ref:`deprecation warnings <iris_deprecations>`, then it
+*could* behave differently, or fail, at the next major release.
+
+
+
+User Actions : How you should respond to changes and releases
+=============================================================
+
+Checklist :
+
+* when a new **testing or candidate version** is announced
+    if convenient, test your working legacy code against it and report any problems.
+
+* when a new **minor version is released**
+
+    * review the 'Whatsnew' documentation to see if it introduces any
+      deprecations that may affect you.
+    * run your working legacy code and check for any deprecation warnings,
+      indicating that modifications may be necessary at some point
+    * when convenient :
+
+      * review existing code for use of deprecated features
+      * rewrite code to replace deprecated features
+
+* when a new major version is **announced**
+    ensure your code runs, without producing deprecation warnings, in the
+    previous minor release
+
+* when a new major version is **released**
+    check for new deprecation warnings, as for a minor release
+
+
+Details
+=======
+
+The Iris change process aims to minimise the negative effects of change, by
+providing :
+
+    * defined processes for release and change management
+    * release versioning
+    * backwards code compatibility through minor version releases
+    * a way to ensure compatibility with a new major version release
+    * deprecation notices and warnings to highlight all impending changes
+
+Our practices are intended be compatible with the principles defined in the
+`SemVer project <http://semver.org/>`_ .
+
+Key concepts covered here:
+    * :ref:`Release versions <iris_change_releases>`
+    * :ref:`Backwards compatibility <iris_backward_compatibility>`
+    * :ref:`Deprecation <iris_deprecations>`
+
+
+.. _iris_backward_compatibility:
+
+Backwards compatibility
+-----------------------
+
+"Backwards-compatible" changes are those that leave all possible existing API
+usages unchanged (see :ref:`terminology <iris_api>` below).
+Only such changes may be included in minor releases.
+
+the following are examples of backward-compatible changes :
+
+    * changes to documentation
+    * adding to a module : new submodules, functions, classes or properties
+    * adding to a class : new methods or properties
+    * adding to a function or method : new optional arguments or keywords
+
+The following are examples of **non-** backward-compatible changes :
+
+    * removing a call
```

Comment:
Should this be "removing a function"? I don't see how removing a call is necessarily backwards-incompatible, unless it's to some user-defined callback.